### PR TITLE
fix some docstring typos

### PR DESCRIPTION
**Pull request by jeis2497052**
are these OK?

### CODE REVIEW

**Comment by basil-conto** [comment]
> are these OK?

LGTM.

**Comment by basil-conto** [comment]
Actually, I would reword the commit message as "Fix some docstring typos".

**Comment by jeis2497052** [comment]
should I amend the git commit ?

**Comment by jeis2497052** [comment]
did I amend the git commit string properly ?

**Comment by basil-conto** [comment]
> should I amend the git commit ?

I'm not the owner, so I can't force you, but I think the word "propose" makes no sense in a commit message. The commit was either merged, or it wasn't. It either fixed the typos, or it didn't. :)

**Comment by jeis2497052** [comment]
sorry, I will remove propose then

**Comment by basil-conto** [comment]
@jeis2497052 I'm talking about the message of your git commit 6a0d1fd, not the GitHub PR title.

**Comment by jeis2497052** [comment]
got it, just amended

**Comment by basil-conto** [comment]
Thanks, and sorry for the pedantry. :)

**Comment by jeis2497052** [comment]
I just want to contribute as I like using swiper and each repo has some nuance regarding PR.

**Comment by basil-conto** [comment]
> I just want to contribute as I like using swiper and each repo has some nuance regarding PR.

Sorry, I didn't mean to deter you (I'm not even a maintainer here!). My comment was only intended as a friendly and constructive suggestion to make the life of `swiper`'s maintainers easier.

In general, though, the "nuances" of different FOSS projects aren't completely unjustified. Successfully maintaining these projects is very hard work and so maintainers often set up guidelines for a certain quality standard and for consistency which will make their life easier. See also https://windsooon.github.io/2017/11/23/Why%20we%20never%20thank%20open%20source%20maintainers/.

Anyway, thanks for catching these typos (one of which I'm responsible for)!

**Comment by jeis2497052** [comment]
last thing I want is to contribute to additional work for any maintainer.  I suppose I could have read commits to guess a little better.  I have happy for follow your guidelines.  Good link regarding the life of a maintainer.  I can't image it.

**Comment by abo-abo** [comment]
Thanks. I really should run `ispell` at least once through the whole code base, I have no ieda where all these typeos are coming from:)

**Comment by jeis2497052** [comment]
no worries, I am the worst speller and live by ispell :-)
